plugins/sep/sepsource.cc

```diff
@@ -67,22 +73,29 @@ SepFile SepSource::parseSep(const std::string &file_name) {
       continue;
     }
 
-    if (result[0] != "C" && result[0] != "M" && result[0] != "Y" &&
-        result[0] != "K" && result[0] != "V" && result[0] != "W") {
-      // Unsupported channel
-      warnings += "WARNING: The .sep file defines an unknown channel (not C, "
-                  "M, Y, K, V or W)!\n";
-      continue;
+    // Load the color corresponding to this name
+    auto correctColor = ColorConfig::getInstance().getColorByNameOrAlias(result[0]);
+    CustomColor newColor = CustomColor(result[0], 0.0, 0.0, 0.0, 0.0);
```

Comment:
I guess this should be called `nonExistantColor` ? And its definition could be inside the `if`, so you don't create it for colors that exist. And it could be `const`

Truth be told, after loading the configuration file, all colors could be `const`.

Reply:
Changed. Colors undefined in the json are now not drawn, and their source files are not loaded.